include/dynd/string.hpp

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (C) 2011-15 DyND Developers
+// BSD 2-Clause License, see LICENSE.txt
+//
+
+#pragma once
+
+#include <dynd/callable.hpp>
+#include <dynd/kernels/string_concat_kernel.hpp>
+
```

Comment:
`#include <dynd/kernels/string_concat_kernel.hpp>` should only be in the *.cpp file. That really simplifies the amount of includes visible to the user.